tests/general_indicator_tests/general_indicator_test_helpers.py

```diff
@@ -1,11 +1,19 @@
-from aggregate.all_accessors import accessors
+from aggregate.all_accessors import Accessors
 
+accessors = Accessors()
 
-def get_by_geo():
+
+def get_by_geo(housing_security=False):
+    """housing security parameter is temporary"""
+    if housing_security:
+        accessors_list = accessors.housing_security
```

Comment:
This is outside the scope of this pull request but when I peek inside the Accessor class implementation. I found the distinction between the two properties a little difficult to understand. Please let me know when there is a moment to chat about it. Thanks!

Reply:
I think reviewing your opening comments explained it. So please ignore my questions for now!